charts/pathfinder-rpc-server/values.yaml

```diff
@@ -0,0 +1,54 @@
+# LetsEncrypt certificates
+acmeEmail:                 
+acmeServer:                
```

Comment:
Would it be possible to remove Let's Encrypt configuration from this chart?

These values don't appear to be used anywhere else in this PR.

Reply:
This is an external configuration that's done via the Ingress annotations in the values.

It does not need to work "out of the box" IMHO: Every cluster maintainer has different assumptions or even different ACME providers, if any.

In other words: A cluster issuer would already be present before installing this Helm chart, as a prerequisite/dependency, and as such cannot be part of its distribution.

Reply:
ok, agree